### ISSUE TITLE
Tasks schema reports "block is a required property" when with_flattened is specified

### ISSUE THREAD

**Issue by burn2l** [ansible/ansible-lint#2978]
Other with_X options are accepted.

```
- name: with_flattened
  ansible.builtin.debug:
    msg: "{{ item }}"
  with_flattened: "{{ items }}"
```

- Bug Report

```
ansible-lint 6.12.0 using ansible 2.13.6
```

**Comment by ajinkyau** [comment]
We can see that `with_flattened` has been moved under `community.general` since 2.10.
Correct usage: `community.general.with_flattened`.

**Comment by burn2l** [comment]
Strange ... when I run ansible-lint on:
```
- name: with_flattened
  ansible.builtin.debug:
    msg: "{{ item }}"
  community.general.with_flattened: "{{ items }}"
```
I get:
**parser-error: conflicting action statements: ansible.builtin.debug, community.general.with_flattened**

Note that with_indexed doesn't generate that error (unless I add community.general)

**Comment by ssbarnea** [comment]
@burn2l That is not strange, is expected due to the way schema validation works. If you type two module names in a single task, you will get this kind of behavior.

**Comment by burn2l** [comment]
I'm confused .... with_flattened is an optional keyword, not a module.  The original task I posted is copied directly from the docs.  ansible-lint fails it because the tasks schema reports "block is a required property" ... required by what?

Note that ansible-playbook will run it when imported.

Meanwhile the with_indexed_items example in the docs works fine ... ansible-lint passes it.

So I still claim that the tasks schema is mis-handling the optional keyword 'with_flattened'

**Comment by burn2l** [comment]
Thanks, I see that 6.12.2 has the fix and the with_flattened optional keyword is now accepted.
BUT I was very surprised to discover that my 6.12.1 python package installation was updated last night and now 3 of the schemas are at the 6.12.2 level.  I guess something runs the refresh_schemas method occasionally.  Is there some way to disable this as I need my python environment to remain stable at the level I last installed.

**Comment by ssbarnea** [comment]
If you run it with offline flag, it will not update the embedded schemas.
